src/pages/dev/send-tokens/interchain-tokens/roadmap.mdx

```diff
@@ -0,0 +1,22 @@
+# ITS Roadmap
+
+## Background
+
+Prior to the release of the Axelar [Interchain Token Service](https://github.com/axelarnetwork/interchain-token-service) (ITS), Axelar allowed for only the transfer of gateway tokens between chains. These gateway tokens are manually registered in Axelar’s consensus the list of these gateway tokens can be found [here](https://docs.axelar.dev/resources/contract-addresses/mainnet#assets). With the release of ITS, now any team can provide multichain functionality to their token, whether its deploying a fresh new token on multiple chains or wiring up a pre deployed token to ITS to provide bridging functionality for the token. Unlike gateway tokens, ITS tokens are not minted on the Axelar chain itself rather they are sent between chains via Axelar [General Message Passing (GMP)](https://docs.axelar.dev/dev/general-message-passing/overview). 
```

Comment:
Not accurate, you could always do this via GMP, we just didn't offer a service to do it for you. Manually is a scary word, by whom? Was this a decentralized solution? Maybe not so much depth here unless it's relevant?

Reply:
Yes good point will reword that first sentence. 

Regarding your second point, From what I understand gateway token additions were not a centralized solution. Did not want to write by the axelar team as I believe this goes against writing guidelines so stuck with _manually_. Happy to change if you have a different term you prefer?